classes/Form/ConfigurationForm.php

```diff
@@ -111,6 +103,15 @@ public function generate()
                             'label' => $this->module->trans('No', [], 'Modules.GAnalytics.Admin'),
                         ],
                     ],
+                    'desc' => $this->module->trans('Universal analytics will stop processing data on July 1, 2023. We recommend switching to Google Analytics 4 as soon as possible.', [], 'Modules.GAnalytics.Admin'),
```

Comment:
I'm adding the `waiting for wording` label since this is a new wording...  But I'm not sure it's that necessary @matshir ?